atrium/vestibulum/trcdb/trccertmgmtbase/trccertmgmtbase.go

```diff
@@ -61,11 +55,19 @@ func CommonMain(certPathPtr *string, driverConfig *eUtils.DriverConfig, mod *kv.
 		return err
 	}
 
-	resourceGroupName := apimConfigMap["RESOURCE_GROUP_NAME"]
-	serviceName := apimConfigMap["SERVICE_NAME"]
+	resourceGroupName, exists := apimConfigMap["RESOURCE_GROUP_NAME"]
+	if !exists {
+		return errors.New("RESOURCE_GROUP_NAME is not populated in apimConfigMap")
+	}
+
+	serviceName, exists := apimConfigMap["SERVICE_NAME"]
```

Comment:
Composite these two lines into one on an if check... just do the assign and check all at once.